src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -34,154 +34,234 @@
   private String userStaff = credentials.userNameStaff;
   private String userStaffPassword = credentials.passwordStaff;
 
-  private ChatPageObject openChatForUser(String userName, String password) {
+  private final String CURRENT_BROWSER_TAB = driver.getWindowHandle();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "CURRENT_BROWSER_TAB" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)